parsers/IN_WE.py

```diff
@@ -18,153 +18,144 @@
 from parsers.lib.config import refetch_frequency
 from parsers.lib.exceptions import ParserException
 
-IN_WE_PROXY = "https://in-proxy-jfnx5klx2a-el.a.run.app"
-EXCHANGE_URL = f"{IN_WE_PROXY}/InterRegionalLinks_Data.aspx/Get_InterRegionalLinks_Region_Wise?host=https://www.wrldc.in"
-CONSUMPTION_URL = f"{IN_WE_PROXY}/OnlinestateTest1.aspx/GetRealTimeData_state_Wise?host=https://www.wrldc.in"
+"""
+This module is used to read the exchange values on a hourly basis for a day from India West to North, East, South. 
+This module also calculates the total electricity consumption by India West.
+The data is fetched from https://www.wrldc.in using APIs which provide data in JSON format
+India West constitutes of Gujrat, Madhya Pradesh, Chattishgarh, Maharashtra, Goa, DD, DNH
 
-EXCHANGES_MAPPING = {
-    "WR-SR": "IN-SO->IN-WE",
-    "WR-ER": "IN-EA->IN-WE",
```

Comment:
I reversed this mapping because we simply need to get records in data which match to WR-SR or WR-ER or WR-NR

- In the older version we were joining the data with this mapping to add a new column which represents sortedZoneKey
+ In the newer version instead of performing a join, I am simply filtering by region name like WR-SR, WR-ER, WR-NR

It is the same filtering logic, just easier to read